src/coreComponents/physicsSolvers/fluidFlow/SinglePhaseBase.cpp

```diff
@@ -490,7 +490,7 @@ void SinglePhaseBase::accumulationLaunch( localIndex const targetIndex,
   arrayView2d< real64 const > const dPvMult_dPres =
     solid.getReference< array2d< real64 > >( ConstitutiveBase::viewKeyStruct::dPVMult_dPresString );
   arrayView1d< real64 const > const bulkModulus =
-    ISPORO ? solid.getReference< array1d< real64 > >( "BulkModulus" ) : porosityOld;
+    ISPORO ? solid.getReference< array1d< real64 > >( "bulkModulus" ) : porosityOld;
```

Comment:
string key?

Reply:
I may leave this as is and fix in another PR.  To add the key I'll have to include a header file for the elastic models, which seems very odd.  I think we need a more careful rethink of how we package the porosity models.